web/src/components/Layout.tsx

```diff
@@ -156,9 +149,10 @@ const Layout = ({
   disableScrollingSafari = false,
 }: LayoutProps): JSX.Element => {
   const modalVisible = !!modal
-  const { width } = useWindowDimensions()
+  const { width, viewportSmall } = useWindowDimensions()
   const [languageSelectorHeight, setLanguageSelectorHeight] = useState<number>(0)
 
+  // TODO fix wrong panel height when switching from local to tunews
```

Comment:
i really couldn't figure out why the `languageSelector` is getting null only on this route. Maybe some can help me out :)

Reply:
The reason for this is that the language change path is null while tu news are loading. Therefore the language selector is not rendered. Therefore, you should use the defaultLanguageChangePath while loading

Reply:
@sarahsporck is right, as long as there is `languageChangePaths` are null, no language selector is rendered and therefore the height is undefined. Once the language selector is rendered, no more rerender/useEffect of the Layout is triggered and therefore the height is never updated. I created a fix in https://github.com/digitalfabrik/integreat-app/pull/955. Automatic retriggering the useEffect would probably be better but not sure how that could work.